src/cli/bootstrap/bootstrap.ts

```diff
@@ -79,9 +95,25 @@ export default async function bootstrap(args: string[]) {
         restoreCreateAccountFromMock();
 
         console.log('Change the key of the accounts to the new key', newPublicKey.toString());
-        await updateAccountKey('id.tonomy', newPublicKey, true);
-        await updateAccountKey('eosio.token', newPublicKey, true);
-        await updateAccountKey('eosio', newPublicKey);
+        await updateAccountKey('found.tmy', newPublicKey, true);
+        // accounts controlled by found.tmy
+        await updateControllByAccount('gov.tmy', 'found.tmy', true);
+        await updateControllByAccount('team.tmy', 'found.tmy', true);
+        await updateControllByAccount('prod1.tmy', 'found.tmy', true);
+        await updateControllByAccount('prod2.tmy', 'found.tmy', true);
+        await updateControllByAccount('prod3.tmy', 'found.tmy', true);
+        //accounts controlled by gov.tmy
+        await updateControllByAccount('eosio', 'gov.tmy');
+        await updateControllByAccount('eosio.token', 'gov.tmy');
+        await updateControllByAccount('id.tmy', 'gov.tmy', true);
```

Comment:
L100:104 & L108 has `addCodePermission = true` and others `false`. What's the reason for this?

Reply:
if added true to those accounts it throws error [
  {
    message: 'Invalid authority: {"threshold":1,"keys":[],"accounts":[{"permission":{"actor":"gov.tmy","permission":"active"},"weight":1},{"permission":{"actor":"eosio","permission":"eosio.code"},"weight":1}],"waits":[]}',
    file: 'eosio_contract.cpp',
    line_number: 264,
    method: 'apply_eosio_updateauth'
  }
] i guess there is weight issue as we controll gov.tmy by found.tmy and others control by gov.tmy

Reply:
What about L100:104 - when and what error is show ( provide full stack trace)
and if you change only L108 - when and what error is shown (provide full stack trace)

Reply:
chnage      await updateControllByAccount('ecosystm.tmy', 'gov.tmy');
to         await updateControllByAccount('ecosystm.tmy', 'gov.tmy', true); getting this error AntelopePushTransactionError: Internal Service Error
    at transact (/home/sadia/tonomy/new/Tonomy-ID-Integration/Tonomy-ID-SDK/build/cli/index.cjs:1437:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async EosioContract.updateauth (/home/sadia/tonomy/new/Tonomy-ID-Integration/Tonomy-ID-SDK/build/cli/index.cjs:1804:12)
    at async updateControllByAccount (/home/sadia/tonomy/new/Tonomy-ID-Integration/Tonomy-ID-SDK/build/cli/index.cjs:3683:3)
    at async bootstrap (/home/sadia/tonomy/new/Tonomy-ID-Integration/Tonomy-ID-SDK/build/cli/index.cjs:4183:5)
    at async main (/home/sadia/tonomy/new/Tonomy-ID-Integration/Tonomy-ID-SDK/build/cli/index.cjs:4213:5) {
  code: 500,
  error: {
    code: 3050000,
    name: 'action_validate_exception',
    what: 'Action validate exception',
    details: [ [Object] ]
  }
}
[
  {
    message: 'Invalid authority: {"threshold":1,"keys":[],"accounts":[{"permission":{"actor":"gov.tmy","permission":"active"},"weight":1},{"permission":{"actor":"ecosystm.tmy","permission":"eosio.code"},"weight":1}],"waits":[]}',
    file: 'eosio_contract.cpp',
    line_number: 264,
    method: 'apply_eosio_updateauth'
  }
] if i didn't add true it works fine